bookmarks.js

```diff
@@ -1,3 +1,21 @@
+import { getSections } from './styles/lib/db.js'
+import { createCard } from './styles/lib/card.js'
+
+const content = document.querySelector('.container__content')
+
+const data = getSections()
+
+const filteredCards = createCard.filter((bookmark) => {
+
+  if(bookmark === 'black') {
+    
+    for (let i = 0; i < data.length; i++) {
+    const cardSection = createCard(data[i])
+    content.append(cardSection)
+  }
```

Comment:
Try to change this evil `for loop` with the new tools you've learned (`array methods`)